src/streaming/models/CmsdModel.js

```diff
@@ -196,11 +196,11 @@ function CmsdModel() {
         // Ge in reverse order in order to consider only last CMSD-Dynamic header
         for (let i = headers.length - 1; i >= 0; i--) {
             const header = headers[i];
-            let m = header.match(/^(?<key>[^:]*):\s*(?<value>.*)$/);
-            if (m && m.groups) {
+            let m = header.match(/^([^:]*):\s*(.*)$/);
+            if (m && m[1] && m[2]) {
```

Comment:
If the Regex doesn't match, `match` returns null, so it's probably sufficient to check `m` is truthy